src/components/App/App.js

```diff
@@ -7,11 +7,12 @@
 
 import React from 'react';
 import { Provider } from 'react-redux';
-import { BrowserRouter, Route } from 'react-router-dom';
+import { Router, Route } from 'react-router-dom';
 
 import store from '../../state/store';
 import janusApi from '../../janus-api';
 import { addEventsHandlers } from './events-handler';
+import history from '../../utils/history';
```

Comment:
Just for the record, probably we could change this in the future if `Buttons` can be `Link` or `NavLink` an the history can be sent in the payload of the action. 

See https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/guides/redux.md